docs/source/release_notes.rst

```diff
@@ -10,6 +10,7 @@ Release Notes
         * Added multiclass check to ``InvalidTargetDataCheck`` for two examples per class :pr:`1596`
         * Support graphviz 0.16 :pr:`1657`
         * Enhanced time series pipelines to accept empty features :pr:`1651`
+        * Add KNN Classifier to baseline. :pr:`1650`
```

Comment:
I think it'd be better to say `Add KNN Classifier to estimators` rather than to baseline?

Reply:
Yup, thank you sir